unfurl/build.gradle.kts

```diff
@@ -1,15 +1,76 @@
 plugins {
-  id("java-library")
-  kotlin("jvm")
+    alias(libs.plugins.kotlinMultiplatform)
+    alias(libs.plugins.androidLibrary)
+    alias(libs.plugins.dokka)
+    id("maven-publish")
 }
-apply(plugin = "com.vanniktech.maven.publish")
 
-dependencies {
-  api("com.squareup.okhttp3:okhttp:4.9.0")  // Updating to 5.x? See cli/build.gradle.kts.
-  api("org.jsoup:jsoup:1.16.1")
+group = "me.saket.unfurl"
+version = libs.versions.libraryVersion.get()
 
-  testImplementation("junit:junit:4.13.2")
-  testImplementation("com.google.truth:truth:1.1.3")
-  testImplementation("com.google.testparameterinjector:test-parameter-injector:1.8")
-  testImplementation("com.squareup.okhttp3:mockwebserver:4.10.0")
+@OptIn(org.jetbrains.kotlin.gradle.ExperimentalKotlinGradlePluginApi::class)
+kotlin {
+    explicitApi()
+
+    jvm()
+
+    androidTarget {
+        compilations.all {
+            kotlinOptions {
+                jvmTarget = "17"
+            }
+        }
+    }
+    
+    listOf(
+        iosX64(),
+        iosArm64(),
+        iosSimulatorArm64()
+    ).forEach {
```

Comment:
is `ksoup` ready for iOS?

Reply:
We need to run test cases to confirm. The library does have support.
However running test cases has a few blockers as mentioned in description